dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/bundle/persister/AbstractTrackerPersister.java

```diff
@@ -198,6 +212,12 @@ public TrackerTypeReport persist(EntityManager entityManager, TrackerBundle bund
   /** Get Tracked Entity for enrollments or events that have been updated */
   protected abstract String getUpdatedTrackedEntity(V entity);
 
+  /** Clones the event properties that may potentially be change logged */
+  protected V cloneEntityProperties(TrackerPreheat preheat, T trackerDto) {
+    return null;
```

Comment:
would it be better to throw an exception like:
`throw new UnsupportedOperationException("Cloning not supported for this entity type.");`
?